builder/bundles.go

```diff
@@ -371,7 +372,8 @@ func resolvePackages(numWorkers int, set bundleSet, packagerCmd []string, emptyD
 			// status. This exit status is 1, which is the same as any other dnf install
 			// error. Fortunately if this is a different error than we expect, it should
 			// fail in the actual install to the full chroot.
-			outBuf, _ := helpers.RunCommandOutput(queryString[0], queryString[1:]...)
+			timeout := 0
+			outBuf, _ := helpers.RunCommandOutput(timeout, queryString[0], queryString[1:]...)
```

Comment:
again, don't see the point of declaring a variable to immediately use and discard

Reply:
People (including me) hated on using magic numbers in these kinds of cases. Was just trying to make it more clear, but I'm thinking of how to re-write.